controller/deploy_controller.rb

```diff
@@ -4,21 +4,28 @@
 require_relative '../helper/user_comms_helper'
 
 class DeployController
-  def initialize(environ, git_helper, user_comms, develop_tag_generator, other_tag_generator)
-    @environ = environ
+
+  attr_accessor :environ
+  attr_accessor :tag_types
+
+  def initialize(environ, git_helper, user_comms, develop_tag_generator, other_tag_generator, tag_types)
+    self.environ = environ
+    self.tag_types = tag_types
```

Comment:
You could do that for the other 4 as well and you can write them all into one `attr_accessor` statement:

`attr_accessor :environ, :tag_types`